lib/active_zuora/generator.rb

```diff
@@ -29,6 +29,7 @@ def generate_classes
           class_name = complex_type.attribute("name").value
           # Skip the zObject base class, we define our own.
           next if class_name == "zObject"
+          class_name[0] = class_name[0].upcase
```

Comment:
I passed on capitalize because it has a 'side-effect' of downcasing the rest of the string.  I think we should keep the capitalization provided by the wsdl where possible since the names are case sensitive.